src/zone/schedule-phase.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include <stdint.h>
+
+namespace napa {
+namespace zone {
+
+    // Define the phase (like priority related type) of tasks. 
+    // To be used mainly by schduler and worker to schedule its tasks.
+    enum class SchedulePhase : uint32_t {
+        kDefaultPhase = 0,
```

Comment:
remove the prefix k for enumerable definition

Reply:
I saw this style in code. Where is the guide line?

Reply:
fixed.